src/modules/interp_met/WindNinja.cpp

```diff
@@ -90,33 +90,22 @@ void WindNinja::init(mesh& domain)
     N_windfield = 0;
     for(auto& itr: domain->parameters() )
     {
-        if( itr.find("Ninja") != std::string::npos)
+        if( itr.find("Ninja") != std::string::npos &&
+            itr.find("_") == std::string::npos) // ignore all _U, _V, and _LAvg params
```

Comment:
I am not sure that this way of counting N_windfield would work in any situations. For example, what would happen if you have only speed up maps defined with _Lavg. This loop would return N_windfield=0.

Reply:
Good point. I now detect _U (arbitrary choice) and use that. 
I also check that _U and _V exist
commit: 85638469df62cd964b5f869e9b6455ab1f989a37